packages/react-component-library/src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,2 @@
+export * from './Dropdown'
```

Comment:
Can you explain the thinking behind this new way of handling the component exports?

Reply:
It's the recommended  way to do exports in TS. It re-exports all the things exported from a thing and also advises not to use default exports.

So you have

Thing.ts

const export type ThingProps {
  name: string
}

export const Thing:React.FC<ThingProps> Thing = ({ name }) => (<p>{name}</p>)


------
Components 
export * from './Thing' 

-------

import { Thing, ThingProps } from './components'

or even 

import { Thing, ThingPros} from './components/Thing'



I started using the pattern and find it easier for things like what we are doing.

Reply:
Ok cool - i'll adopt this pattern going forward.